tests/appsec/appsec/test_remoteconfiguration.py

```diff
@@ -1004,14 +1002,14 @@ def test_rc_rules_data(tracer):
 
 
 def test_rc_rules_data_error_empty(tracer):
-    with override_global_config(dict(_asm_enabled=True)), override_env({APPSEC.ENV: "true"}):
+    with override_env({APPSEC.ENV: "true"}), override_global_config(dict(_asm_enabled=True)):
         tracer.configure(appsec_enabled=True, api_version="v0.4")
         config = {}
         assert not _appsec_rules_data(config, tracer)
 
 
 def test_rc_rules_data_error_ddwaf(tracer):
-    with override_global_config(dict(_asm_enabled=True)), override_env({APPSEC.ENV: "true"}):
+    with override_env({APPSEC.ENV: "true"}), override_global_config(dict(_asm_enabled=True)):
```

Comment:
Why the order change?

Reply:
Because the config is now set at start and env var are checked only at start.
So if you change the environ in test, you need to change the config **after** to update it.